_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -149,14 +149,16 @@ function page_admin_scripts() {
 			? json_decode( file_get_contents( $script_deps_path ) )
 			: array();
 		$script_dependencies[] = 'wp-polyfill';
+		if ( Jetpack::is_active() ) {
```

Comment:
To fix the issue with Development mode, I'd recommend doing something like:

```suggestion
		if ( Jetpack::is_active() || Jetpack::is_development_mode() ) {
```